bitwarden_license/src/Commercial.Core/SecretsManager/Commands/AccessPolicies/CreateAccessPoliciesCommand.cs

```diff
@@ -39,7 +71,7 @@ public async Task<List<BaseAccessPolicy>> CreateAsync(List<BaseAccessPolicy> acc
                 throw new BadRequestException("Resource already exists");
             }
         }
-
-        return await _accessPolicyRepository.CreateManyAsync(accessPolicies);
+        await _accessPolicyRepository.CreateManyAsync(accessPolicies);
+        return await _accessPolicyRepository.GetManyByProjectId(projectId);
```

Comment:
Any reason we need to re-fetch them?

Reply:
`await _accessPolicyRepository.CreateManyAsync(accessPolicies);` just returns what was created. 

Since this endpoint is creating for projects.  `[HttpPost("/projects/{id}/access-policies")]` 
It made more sense from an API perspective and consuming it from the web project that the whole project access policies view was returned.

I'm open to changing it if required.